release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -77,6 +77,10 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
 [id="ocp-4-8-web-console"]
 === Web console
 
+[id="ocp-4-8-custom-console-routes-use-custom-domains-cluster-api"]
+==== Custom console routes now use the new `CustomDomains` cluster API
```

Comment:
I can no longer recall where we landed on markup in titles :sweat_smile:  - is `CustomDomains` ok or does it need no ticks?

Reply:
Good catch! I removed the ticks. Thanks!